example/wildcat.config.js

```diff
@@ -1,7 +1,41 @@
+const fs = require("fs");
+const path = require("path");
+
 const pkg = require("./package.json");
 const __DEV__ = (process.env.NODE_ENV === "development");
 const __PROD__ = (process.env.NODE_ENV === "production");
 const __BUNDLE__ = process.env.COVERAGE !== "e2e";
+const __IN_DOCKER__ = fs.existsSync("/.dockerenv");
+
+
+function getDefaultSSLFile(filename) {
+    const filePath = __IN_DOCKER__ ? "ssl/example." : "../packages/react-wildcat/ssl/server.";
```

Comment:
This seems very fragile. What if I have a `.dockerenv` file but also run `npm run dev` outside of Docker? This file has access to your current hostname, you should be able to set up certificates based on pathname. e.g.:

```
- ssl
  - example.dev
    - server.key
    - server.cert
    - server.ca
  - localhost
    - server.key
    - server.cert
    - server.ca
```